gravitee-am-repository/gravitee-am-repository-jdbc/src/main/java/io/gravitee/am/repository/jdbc/management/api/JdbcIdentityProviderRepository.java

```diff
@@ -50,6 +50,7 @@
     public static final String COL_ID = "id";
     public static final String COL_TYPE = "type";
     public static final String COL_NAME = "name";
+    public static final String COL_SOURCE = "source";
```

Comment:
* Using "system" or "is_system" as value and COL_SYSTEM as constant name will be more appropriate.
* I don't see the liquibase update for this column addition. We use liquibase to update the DB schema when the Management API startup (look at [gravitere-am-repository](https://github.com/gravitee-io/gravitee-access-management/tree/master/gravitee-am-repository/gravitee-am-repository-jdbc/src/main/resources/liquibase) for examples.